connectors/youtube.js

```diff
@@ -124,7 +144,19 @@ function setupMaterialPlayer() {
 	};
 
 	Connector.isStateChangeAllowed = function() {
-		return $('.videoAdUi').length === 0;
+		if ($('.videoAdUi').length > 0) {
+			return false;
+		}
+
+		const params = new URLSearchParams(document.location.search.substring(1));
+		const videoId = params.get('v');
```

Comment:
Video ID is also accessible via `Connector.getUniqueID` function. I think it should be used instead of this.